assets/js/phoenix_live_view.js

```diff
@@ -19,6 +19,7 @@ const PHX_VIEW = "data-phx-view"
 const PHX_COMPONENT = "data-phx-component"
 const PHX_LIVE_LINK = "data-phx-link"
 const PHX_LINK_STATE = "data-phx-link-state"
+const PHX_REF = "data-phx-ref"
```

Comment:
Do we need to put it as a data attribute or can we just keep it in the DOM? I have no preference for one or the other, just curiosity. :)

Reply:
we need it in the DOM attribute so it gets brought over for phx-update=append|prepend